compiler/definitions/ir/aggregator_node.py

```diff
@@ -17,36 +20,73 @@ def __init__(self, old_node, input_ids, output_ids, name_string, new_options):
         super().__init__(input_ids,
                          output_ids, 
                          name,
-                         com_category, 
-                         com_options=old_node.com_options, 
+                         com_category,
+                         # BEGIN ANNO
```

Comment:
Now that you are working with this codebase for a while, we should gradually stop commenting out code and just remove it.

Reply:
Ok, I'll do this. I also removed some commented code from the current PR.